rmgpy/molecule/fragment.py

```diff
@@ -32,32 +32,38 @@
 
 import rmgpy.molecule.group as gr
 import rmgpy.molecule.element as elements
+from rmgpy.molecule.element import Element
 import rmgpy.molecule.converter as converter
 from rmgpy.molecule.element import get_element
 from rmgpy.molecule.graph import Graph, Vertex
 from rmgpy.molecule.molecule import Atom, Bond, Molecule
-from rmgpy.molecule.atomtype import get_atomtype, AtomTypeError, ATOMTYPES
+from rmgpy.molecule.atomtype import get_atomtype, AtomTypeError, ATOMTYPES, AtomType
 from rdkit import Chem
 
 # this variable is used to name atom IDs so that there are as few conflicts by
 # using the entire space of integer objects
 ATOM_ID_COUNTER = -(2**15)
 
 
-class CuttingLabel(Vertex):
+class CuttingLabel(Atom):
     def __init__(self, name="", label="", id=-1):
-        Vertex.__init__(self)
-        self.name = name  # equivalent to Atom element symbol
-        self.label = label  # equivalent to Atom label attribute
-        self.charge = 0
-        self.radical_electrons = 0
-        self.lone_pairs = 0
+        super().__init__(
+            element=Element(0, name, "cutting label", 0.0, -1),
+            radical_electrons=0,
+            charge=0,
+            label=label,
+            lone_pairs=0,
+            site="",
+            coords=None,
+            id=id,
+            props=None,
+        )
+        # define a new custom atom type
+        self.atomtype = AtomType(label=label, generic=[], specific=[])
+        # accessed by other methods in Fragment
+        self.name = name
         self.isotope = -1
-        self.id = id
-        self.mass = 0
-        self.site = ""
-        self.morphology = ""
-
```

Comment:
Why are these added in previous commit but deleted here?

Reply:
I changed inheritance schemes a couple times, will clean up commit history closer to merging.